packages/office-ui-fabric-react/src/components/Shimmer/ShimmerGap/ShimmerGap.types.ts

```diff
@@ -26,11 +26,6 @@ export interface IShimmerGapProps extends React.AllHTMLAttributes<HTMLElement> {
    */
   width?: number | string;
 
-  /**
-   * Sets custom styling of the gap.
-   */
-  borderStyle?: IStyleSet;
```

Comment:
@dzearing are you okay with removing this prop? It is technically a breaking change... 

An alternative is just to change the type to `any` and add `@deprecated`.  (Would be even nicer if we could use TS 3.0's `unknown` type instead for deprecation).

Reply:
Unless it was a blocking bug, this should be @deprecated and not removed.